discovery_child_development/notebooks/00_preprocess_openalex.py

```diff
@@ -0,0 +1,188 @@
+# %% [markdown]
+# This notebook is for preprocessing the Openlex data we have extracted and turning it into a format that can be the input to some text clustering.
+#
+# The data was generated by running `discovery_child_development/pipeline/openalex_workflow.py` from the command line via
+# ```
+# python discovery_child_development/pipeline/openalex_workflow.py run --production=True
+# ```
+#
+# Two outputs are saved:
+# * the larger of the two is a table that includes in long format every concept that is associated with each OpenAlex ID and their scores
+# * the smaller is a table with one row per work, containing the OpenAlex ID, title, abstract (deinverted) and "text" which is simply title + ' ' + abstract.
+
+# %%
+import json
+import os
+import boto3
+from dotenv import load_dotenv, find_dotenv
+import pandas as pd
+from io import StringIO
+
+# %%
+# We need to change wd in order to import utils functions
+env_path = find_dotenv()
+env_dir = os.path.dirname(env_path)
+os.chdir(env_dir)
+os.getcwd()
+
+# Utils functions
+from discovery_child_development.utils import openalex_utils
+
+# %%
+CONCEPT_IDS = [
+    "C109260823",  # child development
+    "C2993937534",  # childhood development
+    "C2777082460",  # early childhood
+    "C2911196330",  # child rearing
+    "C2993037610",  # child care
+    "C2779415726",  # child protection
+    "C2781192327",  # child behavior checklist
+    "C15471489",  # child psychotherapy
+    "C178229462",  # early childhood education
+    # "C138496976",  # developmental psychology (level 1).
+]
```

Comment:
Hey, this is why it's better to keep your list of concepts in one centralised place (see the other comment for `openalex_metflow.py`). 

You can keep it like this for now, but let's implement the centralised concepts in the next PR (I'd suggest adding an issue for that).